src/components/Asset/AssetLogo.tsx

```diff
@@ -0,0 +1,61 @@
+import Image from "next/image"
+import { CSSProperties, FC, useMemo, useState } from "react"
+import { BiErrorCircle } from "react-icons/bi"
+import { Address } from "wagmi"
+
+import clsxm from "@/lib/clsxm"
+import useCurrencyLogoURI from "@/hooks/useCurrencyLogoURIs"
+
+import BalancerLogo from "~/images/assets/balancer.png"
+import CurveLogo from "~/images/assets/curve.png"
+
+type AssetLogoProps = {
+  name?: string
+  className?: string
+  style?: CSSProperties
+  tokenAddress?: Address
+}
+
+export const AssetLogo: FC<AssetLogoProps> = ({
+  className,
+  name,
+  tokenAddress,
+  style,
+}) => {
+  const [isError, setIsError] = useState(false)
+
+  const { logoURI } = useCurrencyLogoURI(tokenAddress || "0x")
+  const isStatic = name === "curve" || name === "balancer"
+  const imageSrc = useMemo(() => {
+    switch (name) {
+      case "curve":
+        return CurveLogo
+      case "balancer":
+        return BalancerLogo
+      case "token":
+        return logoURI
+      default:
+        return ""
+    }
+  }, [name, logoURI])
+
+  return (
+    <div
+      className={clsxm("relative rounded-full bg-white", className)}
```

Comment:
You should never write components that require specific CSS rules on a parent. Updated here to contain the required positioning within the component itself.